content/projects/gsoc/index.adoc

```diff
@@ -100,6 +100,14 @@ In addition to these organization-wide meetings,
 each GSoC project has regular meetings during community bonding and coding phases.
 See the project pages for the schedule.
 
+=== Project Proposal Review
+
+Instead of using common using link:https://gitter.im/jenkinsci/gsoc-sig[GSoC Gitter channel] for technical deep dive
+or project proposal for your project, we suggest using the project specific channel or the mailing list 
+link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[jenkinsci-gsoc-all-public] to ask for questions and reviews.
```

Comment:
Better to reference mailing lists referenced as contacts in project ideas

Reply:
Actually we have already had a `Mailing lists` section, but somehow it's still not effective. I am thinking of copying the content from https://jenkins.io/projects/gsoc/students/ to make email to mailing list as a requirement for GSoC. WDYT?

Reply:
works for me

Reply:
Yes, the mailing list I think is the better place, easier to search and easier to follow threads.

Reply:
sorry for the delay, I have just updated my change. For your review @oleg-nenashev @martinda

Reply:
+1 for the requirement to mail out over the mailing list

Reply:
I would make the mailing list a requirement. It is too hard to keep track of all the chat rooms.